.github/workflows/reusable-macos.yml

```diff
@@ -47,11 +47,23 @@ jobs:
           ${{ inputs.free-threading && '--disable-gil' || '' }} \
           --prefix=/opt/python-dev \
           --with-openssl="$(brew --prefix openssl@3.0)"
+    - name: Build CPython for Compiler Warning Check
+      if : ${{ !inputs.free-threading && inputs.os == 'macos-14' }}
+      run: set -o pipefail; make -j1 2>&1 | tee compiler_output_macos.txt
     - name: Build CPython
-      run: set -o pipefail; make -j8 2>&1 | tee compiler_output.txt
+      if : ${{ inputs.free-threading || inputs.os != 'macos-14' }}
+      run: make -j4
```

Comment:
Why `-j4` here? It was `-j8` before.

Reply:
I was experimenting with this option to see how it affected compiler output. The thread count in the Ubuntu job doesn't seem to impact the ability to write to stdout without malforming the diagnostics so I will revert this to `-j8`

Reply:
Oh never mind this is for the non-warning-checking macOS job. Yes will definitely revert to `-j8`